src/components/map/VehiclePopupContent.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+	export let nextDestination;
+	export let vehicleId;
+	export let lastUpdateTime;
+	export let nextStopName;
+	export let predicted;
+
+	function formatLastUpdated(timestamp) {
```

Comment:
this feels like it should be extracted into the `$lib/formatters.js` file, and it will also need to be internationalized/localized.

Reply:
You are correct; I was unaware of this file.